packages/datetime/src/_daterangepicker.scss

```diff
@@ -18,6 +18,12 @@
     display: none;
   }
 
+  .#{$ns}-daterangepicker-timepickers {
+    display: flex;
+    justify-content: space-around;
+    clear: both;
```

Comment:
@Sam-Kramer why did you add this property? it has no effect since we don't use floats.